src/database/mongodb.ts

```diff
@@ -3,7 +3,7 @@ import dotenv from 'dotenv';
 
 dotenv.config();
 
-const connStr: string = process.env['MONGODB_CONN_STR'] ?? '';
+const connStr: string = process.env['MONGODB_CONN_STR'] ?? 'mongodb+srv://admin:341@cluster0.5rwti.mongodb.net/organize-me';
```

Comment:
This needs to be removed and purged as it's sensitive information. This is a public PR and it should not show this. The reason it's an empty string is because we want it to break if that value is not provided.  https://www.youtube.com/watch?v=z8tIOYg_oho

Reply:
removed and fixed